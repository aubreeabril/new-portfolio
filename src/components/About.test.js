import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import About from "./About";

describe("About", () => {
  it("renders the About section", () => {
    const wrapper = shallow(<About />);
    expect(wrapper.find(".About").length).toEqual(1);
  });
  it("matches its snapshot", () => {
    const section = renderer.create(<About />).toJSON();
    expect(section).toMatchSnapshot();
  });
});
