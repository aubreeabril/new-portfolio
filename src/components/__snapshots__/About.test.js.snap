// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches its snapshot 1`] = `
<div
  className="About"
>
  <div
    className="jumbotron"
    style={
      Object {
        "backgroundColor": "white",
        "minHeight": 768,
      }
    }
  >
    <i
      className="fas fa-code"
      style={
        Object {
          "color": "#d1bfa7",
          "fontSize": "7em",
          "margin": "20px",
          "textAlign": "right",
        }
      }
    />
    <div
      className="text-left gold-card card"
    >
      <p
        className="gold-card-text card-text"
      >
        I'm a full-stack web developer and recovering journalist with experience in
         
        <span
          className="emphasized"
        >
          <strong>
            Ruby on Rails
          </strong>
        </span>
        ,
         
        <span
          className="emphasized"
        >
          <strong>
            JavaScript
          </strong>
        </span>
         
        and
         
        <span
          className="emphasized"
        >
          <strong>
            React
          </strong>
        </span>
        . In my previous life, I worked on a team that was responsible for emailing developers when things were broken and discovered I wanted to be the one making improvements instead.
      </p>
    </div>
  </div>
</div>
`;
